addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -651,11 +651,22 @@ function factory(dependencies) {
             const convertedData = this.env.models['mail.message'].convertData(data);
             const messageIds = this.env.models['mail.message'].all().map(message => message.id);
             const partnerRoot = this.env.messaging.partnerRoot;
-            this.env.models['mail.message'].create(Object.assign(convertedData, {
+            const message = this.env.models['mail.message'].create(Object.assign(convertedData, {
                 author: [['link', partnerRoot]],
                 id: (messageIds ? Math.max(...messageIds) : 0) + 0.01,
                 isTransient: true,
             }));
+            for (const channelId of data.channel_ids) {
+                const thread = this.env.models['mail.thread'].findFromIdentifyingData({
+                    id: channelId,
+                    model: 'mail.channel'
+                });
```

Comment:
maybe can iterate `message.threads` instead of doing this?

Reply:
or just `nonOriginThreads` if we want to strictly match `channel_ids`

Reply:
We don't support commands in non-channels?

Reply:
Or `serverChannels` actually, what a mess all of those fields.

Reply:
Is it really better ? I thought we could just trust the server here ?

Reply:
The scroll part will be fully handled in task-2336251 you can remove it